megamek/src/megamek/client/ratgenerator/ModelRecord.java

```diff
@@ -146,6 +147,21 @@ public boolean isClan() {
         return clan;
     }
 
+    /**
+     * @return true, if unit is base IS tech and mounts Clan tech equipment
+     */
```

Comment:
Are base Clan units with IS equipment not covered by this?

Reply:
There are comparatively few Clan units with IS tech, and with far lower impact, so it doesn't cover those.  This is primarily to hit the early Clan/Clan Invasion refits of classis IS designs with Clan-tech weapons like the Shadow Hawk C, Rifleman C, Warhammer C, and so on.

Maybe change the method name to isMixedISTech, just to be explicit?